kasa/experimental/modules/time.py

```diff
@@ -0,0 +1,91 @@
+"""Implementation of time module."""
+
+from __future__ import annotations
+
+from datetime import datetime, timezone, tzinfo
+from typing import cast
+
+from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+
+from ...cachedzoneinfo import CachedZoneInfo
+from ...feature import Feature
+from ...interfaces import Time as TimeInterface
+from ..smartcameramodule import SmartCameraModule
+
+
+class Time(SmartCameraModule, TimeInterface):
+    """Implementation of device_local_time."""
+
+    NAME = "Time"
+    QUERY_GETTER_NAME = "getTimezone"
+    QUERY_MODULE_NAME = "system"
+    QUERY_SECTION_NAMES = "basic"
+
+    _timezone: tzinfo = timezone.utc
+    _time: datetime
```

Comment:
These are meant probably to be instance-specific?

Reply:
Yes but putting them here saves having to define an `__init__`

Reply:
Yeah, but it'd break if there happens to be two or more devices with different timezones, right?

Reply:
Once they are accessed with `self.` they become instance attributes.